package.json

```diff
@@ -49,6 +45,7 @@
     "enzyme-to-json": "^3.3.0",
     "eslint": "^4.13.1",
     "eslint-config-nava": "^2.1.0",
+    "event-source-polyfill": "^0.0.12",
```

Comment:
This doesn't appear to be used

Reply:
You're right. I must have overlooked removing it from the package.json file while troubleshooting on IE11. I'll create a new branch and remove it later today.